javascript/builtins/String.json

```
@@ -1299,45 +1299,98 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String/match",
             "spec_url": "https://tc39.es/ecma262/#sec-string.prototype.match",
             "support": {
-              "chrome": {
-                "version_added": "1"
-              },
-              "chrome_android": {
-                "version_added": "18"
-              },
+              "chrome": [
+                {
+                  "version_added": "64"
+                },
+                {
+                  "version_added": "1",
+                  "version_removed": "64",
+                  "notes": "No support for named capturing groups."
+                }
+              ],
+              "chrome_android": [
+                {
+                  "version_added": "64"
+                },
+                {
+                  "version_added": "18",
+                  "version_removed": "64",
+                  "notes": "No support for named capturing groups."
+                }
+              ],
               "edge": {
-                "version_added": "12"
+                "version_added": "12",
+                "notes": "No support for named capturing groups."
               },
               "firefox": {
-                "version_added": "1"
+                "version_added": "1",
+                "notes": "No support for named capturing groups."
               },
               "firefox_android": {
-                "version_added": "4"
+                "version_added": "4",
+                "notes": "No support for named capturing groups."
               },
               "ie": {
-                "version_added": "4"
-              },
-              "nodejs": {
-                "version_added": true
-              },
-              "opera": {
-                "version_added": true
-              },
-              "opera_android": {
-                "version_added": true
+                "version_added": "4",
+                "notes": "No support for named capturing groups."
               },
+              "nodejs": [
+                {
+                  "version_added": "10.0.0"
+                },
+                {
+                  "version_added": "0.1.100",
+                  "version_removed": "10.0.0",
+                  "notes": "No support for named capturing groups."
+                }
+              ],
+              "opera": [
+                {
+                  "version_added": "51"
+                },
+                {
+                  "version_added": "15",
```

Almost there!  Based upon our discussion on Opera Android, this should also be `true` since it could be in pre-Blink Opera.  Once we've got that down, I'd say we're ready to go!

```suggestion
                  "version_added": true,
```